flocker/cli/script.py

```diff
@@ -250,9 +254,44 @@ def _changestate_on_nodes(self, deployment, deployment_config,
         return DeferredList(results)
 
 
+@flocker_standard_options
+class CLIOptions(Options):
+    """
+    Command line options for ``flocker`` CLI.
+    """
+    longdesc = """flocker does nothing (yet)."""
```

Comment:
Maybe add reference to `flocker-deploy` as alternative until this actually does stuff.